src/pages/ProductList/test/ProductList.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { waitForElement } from 'react-testing-library'
+import 'jest-styled-components'
+
+import { App } from '../../../App'
+import urls from '../../../constants/urls'
+import { renderWithRouter } from '../../../utilsTest/render'
+import { mockFetchProducts } from '../../../utilsTest/mockHelpers'
+/*import { PRODUCTS } from '../../../utilsTest/mockData'*/
+
```

Comment:
add here `afterEach(cleanup)` (cleanup is from react-testing-library) and test should work fine

Reply:
Fixed!